dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -792,13 +792,20 @@ public List<Tool> allPublishedContainers(
         @ApiParam(value = "Sort column") @DefaultValue("stars") @QueryParam("sortCol") String sortCol,
         @ApiParam(value = "Sort order", allowableValues = "asc,desc") @DefaultValue("desc") @QueryParam("sortOrder") String sortOrder,
```

Comment:
You could change `String sortOrder` to `SortOrder sortOrder`, where there is:

```
enum SortOrder {
   DESC,
   ASC
}
```
That the request wouldn't even get to our code for the example in your bug, which is passing the value `sortOrder=string`. If you did, you should probably do it everywhere (we have `sortOrder` all over), so that could be a new bug.